pages/linux/gsettings.md

```diff
@@ -0,0 +1,32 @@
+# gsettings
+
+> query and modify dconf settings with schema validation
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_the_desktop_environment_in_rhel_8/configuring-gnome-at-low-level_using-the-desktop-environment-in-rhel-8#using-gsettings-command_configuring-gnome-at-low-level>.
+
+- Dump all (non-relocatable) schemas, keys, and values:
```

Comment:
By `dump` what do you mean, does it give complete raw data or is it more 'sincere' in how it gives the info

We normally use Display or Print

Reply:
I have made this change, using "display" where the output is more useful for human consumption, and "print" where the output is directly machine-readable and probably less likely to change and break one's scripts in the future.

Reply:
It is indeed not entirely complete (doesn't contain the output of `range` or `describe`, for example), and not importable by anything AFAIK. But it is useful for piping to grep if you forget the exact name of a schema or key.